openfoam-interface/internal-interface/wrapper/simpleFoam.H

```diff
@@ -0,0 +1,155 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
`openfoam-interface/internal-interface/wrapper/simpleFoam.H` is not used anywhere. It should be removed if that is the case.